recipes/ut/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, CMake, tools
+import os
+import glob
+
+
+class UTConan(ConanFile):
+    name = "ut"
+    description = "C++20 single header/single module, "
```

Comment:
https://github.com/boost-ext/ut/blob/3b05dca6a629497910cf8e92aebcaead0124c8b4/cmake/ut-config.cmake.in#L6

😕

Reply:
Great catch! The description on the GitHub page is a bit out-of-sync with this. I've updated the minimum standard to C++17 and updated the description to reflect support for C++17.